Permanent/Models/Enums/FieldNameUI.swift

```diff
@@ -5,21 +5,22 @@
 //  Created by Lucian Cerbu on 07.12.2021.
 //
 
-import Foundation
-
 enum FieldNameUI: String, CaseIterable {
- 
-    case archiveName = "profile.basic"
-    case shortDescription = "profile.blurb"
-    case longDescription = "profile.description"
-    case emailAddress = "profile.email"
-    case profileGender = "profile.gender"
     
-    var fieldToInsertString: String {
-        switch self {
-        case .archiveName, .shortDescription, .emailAddress, .profileGender: return "string1"
-        case .longDescription: return "textData1"
-        }
-    }
+    case basic = "profile.basic"
+    case blurb = "profile.blurb"
+    case description = "profile.description"
+    case email = "profile.email"
+    case profileGender = "profile.gender"
+    case birthInfo = "profile.birth_info"
+}
 
+struct FieldType {
```

Comment:
is this still in use? I feel like we could probably get rid of it unless we want to use it as a type for the "type" property in ProfileItemModel